api/src/main/java/uk/gov/dvsa/mot/app/TradeServiceRequestHandler.java

```diff
@@ -435,9 +435,8 @@ private String parseVersionNumber(ContainerRequestContext requestContext) {
     }
 
     /**
-     * This method accepts the query string of VRMs (separated by commas) and parses them into
-     * a trimmed set.
-     *
+     * This method accepts the query string of VRMs (separated by commas) and parses them into a trimmed set.
+     * <p>
```

Comment:
Should this tag be here?

Reply:
This tag was introduced by IntelliJ autoformatting using DVSA code style. I noticed that this tag is present in some other classes, e.g. TradeServiceRequestHandlerTest, line 200, that's why I didn't remove it. I'm happy to do so if you prefer.